src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,64 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState, useEffect } from 'react';
+import { Status } from '../../types/Status';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface TodoFilterProps {
+  searchForTodos: (query: string, status: Status) => void,
```

Comment:
checkout checklist pt 3 about props naming 
```suggestion
  onTodosSearch: (query: string, status: Status) => void,
```